design/images/component-ui.puml

```diff
@@ -1,16 +1,58 @@
-@startuml
-node "Seedcase Docker Framework" {
- node "Front End" {
-  node "User Interface" {
-    [Landing page]
-    [User Login pages]
-    [Project list pages]
-    [Project information ages]
-    [Data list pages]
-    [Data information pages]
-    [Upload Data pages]
-    [Download Data pages]
-    [Data QC process Pages]
- }
+@startuml 
+' Component diagram for the Seedcase product - Backend: File Storage
+!include  https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Component.puml
+
+Container(bee, "Back End Environment")
+Container(management, "User and Data Management Layer")
+Container(auth, "Authentication")
+
+Container_Boundary(ui, "Front End User Interface") {
+  Container_Boundary(auth, "Authentication") {
```

Comment:
Should this be a dark blue box (like "Back End Environment" and User and "Data Management Layer") as it is in the first figure on the page ["C4 Container diagram showing the major internal parts of Seedcase."](https://seedcase-project.org/design/software-architecture/building-block-view#level1)?